Xamarin.Forms.Platform.iOS/DisposeHelpers.cs

```diff
@@ -0,0 +1,52 @@
+namespace Xamarin.Forms.Platform.iOS
+{
+	internal static class DisposeHelpers
+	{
+		internal static void DisposeModalAndChildRenderers(this Element view)
```

Comment:
the class is `internal` (no need to specify it, btw), so the member can be `public`. it's slightly less confusing...